client/me/security-account-recovery/recovery-email.jsx

```diff
@@ -20,7 +20,7 @@ class RecoveryEmail extends Component {
 			<ManageContact
 				type="email"
 				isLoading={ isLoading }
-				title={ translate( 'Recovery Email Address' ) }
+				title={ translate( 'Recovery email address' ) }
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.